x-pack/plugins/infra/server/services/inventory_views/inventory_views_client.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  Logger,
+  SavedObject,
+  SavedObjectsClientContract,
+  SavedObjectsUpdateResponse,
+} from '@kbn/core/server';
+import Boom from '@hapi/boom';
+import {
+  staticInventoryViewAttributes,
+  staticInventoryViewId,
+} from '../../../common/inventory_views';
+import type {
+  CreateInventoryViewAttributesRequestPayload,
+  InventoryViewRequestQuery,
+} from '../../../common/http_api/latest';
+import type { InventoryView, InventoryViewAttributes } from '../../../common/inventory_views';
+import { decodeOrThrow } from '../../../common/runtime_types';
+import type { IInfraSources } from '../../lib/sources';
+import { inventoryViewSavedObjectName } from '../../saved_objects/inventory_view';
+import { inventoryViewSavedObjectRT } from '../../saved_objects/inventory_view/types';
+import type { IInventoryViewsClient } from './types';
+
+export class InventoryViewsClient implements IInventoryViewsClient {
+  constructor(
+    private readonly logger: Logger,
+    private readonly savedObjectsClient: SavedObjectsClientContract,
+    private readonly infraSources: IInfraSources
+  ) {}
+
+  static STATIC_VIEW_ID = '0';
```

Comment:
**note**: keep the previous id for the static view always present.
I initially wanted to change it to the `static` string, but as it relies on a value saved in the `infrastructure-ui-source` SO, changing it would imply also writing a migration for that SO, resulting in overengineering a simple rename.